specs/create_Edit_pool.adoc

```diff
@@ -0,0 +1,113 @@
+// vim: tw=79
+
+= Create and Edit Pool
+
+Write a functionality for creating and editing a Pool in Tendrl-Dashboard UI
+
+
+== Problem description
+
+For Creating the new Pool we need to Provide forms with different fields
+* Pool Name
+* Number of Pools to create
+* Type()
+* Replicas
+* Cluster it belongs
+* Journal configration
+* OSD's
+* PG Count
+* Owner
+* Quotas
+
+For editing a Pool we can edit:
+* Pool Name
+* Replicas  
+* PG-count
+* Quotas
+
+== Use Cases
+
+The user should be able to create a new Pool with the UI and also Edit the values of Pool before confirming the pool creation.
+
+== Proposed change
+
+* Create a Form for Creating a new pool.
+* Create a new secondary side nav which will have(https://redhat.invisionapp.com/share/2T9LPMEEB#/screens/209774648)
+  - Basic setting
+  - Advance Configration
+  - Quotas
+* Create a list view For the created pools(https://redhat.invisionapp.com/share/2T9LPMEEB#/screens/217733678)
+* Create review page with data list view.(https://redhat.invisionapp.com/share/2T9LPMEEB#/screens/217733679)
+* List view should have edit button through which we can edit the flowing item listed above.
+
+=== Alternatives
+
+The secondary nav is not available in patternfly so we need to create a new one.
+
+=== Data model impact:
+
+=== Impacted Modules:
+
+==== Tendrl API impact:
+
+Endpoints still to be given for the post calls
+
+==== Notifications/Monitoring impact:
+
+==== Tendrl/common impact:
+
+==== Tendrl/node_agent impact:
+
+==== Sds integration impact:
+
+=== Security impact:
+
+=== Other end user impact:
+
+- End user will be able to create and edit a pool. 
+
+=== Performance impact:
+
+=== Other deployer impact:
+
+=== Developer impact:
+
+=== Implementation:
+
+- Create a directive/component for create pool functionality
+- Create a review page using detail list view of patternfly
+=
+
+
+=== Assignee(s):
+
+Primary assignee:
+  https://github.com/cloudbehl
+
+Other contributors:
+  https://github.com/gnehapk
+  https://github.com/kamleshverma1
+
+=== Work Items:
+
+https://github.com/Tendrl/dashboard/issues/130
+
+== Dependencies:
+https://github.com/Tendrl/dashboard/issues/131
+
+== Testing:
+
+- If create form matches with UI design
+- Next, cancel and Create Buttons are working correctly
+- List view is created after the creatin of Pools
+- You can edit from the list view by clicking on edit button.
```

Comment:
I remember Ju talking about dialog based editing of pools and rbds. @julienlim please confirm.